gradle-baseline-java-config/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -24,4 +24,5 @@
     <suppress files="[/\\].*[/\\]generated_src[/\\]" checks="." />
     <suppress files="[/\\].*[/\\]generated_testSrc[/\\]" checks="." />
     <suppress files="[/\\].*[/\\]generated[/\\]" checks="." />
+    <suppress files="[/\\].*[/\\]generated-schema[/\\]" checks="." />
```

Comment:
I've seen many projects generate their atlas schema code into the `generated` source set -- is there a reason for preferring `generated-schema`?

Reply:
It does not work if the schema itself depends on some generated classes, like some immutables objects.

Reply:
I mean, I think you can make it "work", but in a manual cumbersome way. When generating the schema files, you already need the other depended generated classes to be present. The schema files are generated by a gradle task, not by annotation processing. I've tried setting some proper gradle dependencies over a year ago, but couldn't make it work in those situations (especially when trying to clean up the schema files before re-generating them) and gave up because the setup with an extra folder was just so much simpler.